### ISSUE TITLE
Oathstones jewelry unlock

### ISSUE THREAD

**Issue by Throatripper** [WarEmu/WarBugs#7941]
Im trying to finish the Oathstones jewelry, but when i pickup Tooth Gnoblar's Tooth, it wont update the quest. I have the item in my quest bag, but i cant do anything with it and i cant finish the quest.

For reference
"(54k, 54k) Travel to Thunder Mountain and head far West to Cinderfall and make your way to the coordinates. Slay enough Gutbash Trappers until you find one that drops an item called Gnoblar Trapper's Trap Parts. **In the same area, slay enough Gutbash Tooth Gnoblars until you find one that drops an item called Tooth Gnoblar's Tooth**.

**Comment by Ryanfawkez** [comment]
Cannot reproduce on dev server. Try removing the item and/or retaking the quest altogether.

**Comment by Throatripper** [comment]
Hi! I logged in again last night and the quest now stayed i missed the trapper parts instead of the tooth, wierd. But i had 2 trapper parts and the quest still stated i had not completed that part. I abandoned quest, took it again and still no update. I removed both the teeth and the trapper parts and went back to cinderfall and got the parts. Onscreen i got the message quest was completed for both objectives, however ToK quest part still claimed i didnt have any trapper parts. Even with the quest unfinished according to ToK i went back to lyceum and i could turn the quest in to the IRC and unlock the set rings. All very confusing and annoying but its done now, i managed to finish the quest. Thank you for your time and effort! Best regards //Ripper
